quartz/styles/base.scss

```
@@ -20,12 +20,11 @@ section {
 }
 
 .text-highlight {
-  background-color: #fff23688;
+  background-color: var(--textHighlight);
   padding: 0 0.1rem;
   border-radius: 5px;
 }
-
-::selection {
+S ::selection {
```

```suggestion
::selection {
```

seems like you got an extra S there

---

ah yes, On my macbook I have caps lock rebound to cmd so I will have pressed caps lock + S trying to save and instead just put an `S` in. My bad, I have removed it now.